backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -32,6 +34,24 @@
 public class Translators {
   private static final DateTimeFormatter US_SLASHDATE_SHORT_FORMATTER =
       DateTimeFormatter.ofPattern("M/d/yyyy");
+
+  private static final long LOOK_BACK_YEARS = 99;
+
+  // Accepts either two-digit or four-digit years.
+  // Two-digit years will always resolve to years past.
+  // (e.g., if today is Jan. 1, 2023, and the formatter is passed 11/4/24, it will resolve to Nov 4,
+  // 1924).
+  public static final DateTimeFormatter PAST_DATE_FLEXIBLE_FORMATTER =
```

Comment:
I'm a little confused about why this parser is created but I don't see it being used anywhere besides the validation. shouldn't it also be used to parse the date when creating the patient?

Reply:
@mehansen you are exactly correct, I noticed the same thing this morning 😅 I originally changed the SHORT_DATE_FORMATTER which did affect the parsing on save, but since I didn't want to inadvertently break any other parts of the application switched to a new formatter. But of course that doesn't help when we parse the date now 😄 Going to change this and push up a new commit shortly!